packages/suite-desktop-core/src/electron-react-devtools.ts

```diff
@@ -0,0 +1,28 @@
+import { app, session } from 'electron';
+import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-installer';
+
+app.whenReady().then(() => {
+    session.defaultSession.webRequest.onBeforeRequest((details, callback) => {
+        const allowedUrlPatterns: RegExp[] = [
+            /^https?:\/\/clients2\.google\.com\/.*$/i,
+            /^https?:\/\/update\.electronjs\.org\/.*$/i,
+            /^https?:\/\/localhost(?::\d+)?\/.*$/i,
+            /^https?:\/\/127\.0\.0\.1(?::\d+)?\/.*$/i,
+            /^devtools:\/\//i,
+            /^chrome-extension:\/\//i,
+            /^file:\/\//i,
+        ];
+
+        const urlStr = details.url;
+        const isAllowed = allowedUrlPatterns.some(rx => rx.test(urlStr));
+
+        return callback({ cancel: !isAllowed });
+    });
+
+    installExtension(REACT_DEVELOPER_TOOLS, { loadExtensionOptions: { allowFileAccess: true } })
+        .then(() => {
+            // NOTE: for some reasone, you first need to open dev tools and reload the app to see them
```

Comment:
This really sucks, don't know how to fix it. 

I could manually open dev tools, but that may be bothering for the devs.